sdk/loadtestservice/Azure.Developer.LoadTesting/src/TestRunOperation.cs

```diff
@@ -103,18 +125,41 @@ public override async ValueTask<Response> UpdateStatusAsync(CancellationToken ca
                 return GetRawResponse();
             }
 
-            _response = await _client.GetTestRunAsync(_testRunId).ConfigureAwait(false);
-            _value = _response.Content;
+            try
+            {
+                _response = await _client.GetTestRunAsync(_testRunId).ConfigureAwait(false);
+                _value = _response.Content;
+            }
+            catch
+            {
+                throw new RequestFailedException(_response);
+            }
```

Comment:
I don't think you need this try/catch block since `_client.GetTestFileAsync` will throw RFE if it fails